mkosi/resources/mkosi-vm/mkosi.conf

```diff
@@ -1,5 +1,8 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
+[Match]
+Architecture=uefi
```

Comment:
We should fail loudly if trying to use this on non-uefi, not silently skip the entire thing, that's way worse to debug

Reply:
I'd just drop this commit

Reply:
but the image doesn't build without this, as mkosi-vm is pulled in to build the images, so it fails

Reply:
And now it'll build and won't actually boot in a VM, even though the profile is called mkosi-vm. What good is a mkosi-vm profile if including it doesn't actually make the image bootable in a VM.

If you want this not fail, rework the mkosi-vm configuration to not enable the BiosBootloader= and Bootloader= options when building for the relevant architectures

Reply:
Ok that will need more testing, I've dropped this commit from this PR for now, will be in a future one